daemon/api.go

```diff
@@ -967,6 +976,119 @@ func deleteCapability(c *Command, r *http.Request) Response {
 	}
 }
 
+// skillGrant is a part of response from getSkills().
+type skillGrant struct {
+	Snap string `json:"snap"`
+	Name string `json:"name"`
+}
+
+// skillInfo is a part of response from getSkills().
```

Comment:
`// skillInfo holds details for a skill as returned by the REST API.`

(fixing these comments was part of the last review, fwiw)